README.md

```diff
@@ -281,7 +281,7 @@ These rules are purely matters of style and are quite subjective.
 |:x:|[sort-vars](http://eslint.org/docs/rules/sort-vars)|sort-vars|sort variables within the same declaration block|
 |:x:|[space-before-blocks](http://eslint.org/docs/rules/space-before-blocks)|space-before-blocks|require or disallow a space before blocks|
 |:x:|[space-before-function-paren](http://eslint.org/docs/rules/space-before-function-paren)|space-before-function-paren|require or disallow a space before function opening parenthesis|
-|:x:|[space-in-parens](http://eslint.org/docs/rules/space-in-parens)|space-in-parens|require or disallow spaces inside parentheses|
+|:x:|[space-in-parens](http://eslint.org/docs/rules/space-in-parens)|[space-in-parens](https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/spaceInParensRule.md)|require or disallow spaces inside parentheses|
```

Comment:
I just noticed that this was not updated properly since it still says that its unavailable (`:x:`). To make sure this is updated properly you'll have to edit `src/readme/rules.ts`. Change from:

```ts
  {
    available: false,
    eslintRule: 'space-in-parens',
    tslintRule: 'space-in-parens',
    category: 'Stylistic Issues',
    description: 'require or disallow spaces inside parentheses',
    eslintUrl: 'http://eslint.org/docs/rules/space-in-parens',
    provider: 'tslint-eslint-rules',
    usage: `~~~json
    "space-in-parens": [
        true,
        "always"
      ]
    ~~~
    
    ~~~json
    "space-in-parens": [
        true,
        "never"
      ]
    ~~~`
  },
```

to

```ts
  {
    available: true,
    eslintRule: 'space-in-parens',
    tslintRule: 'space-in-parens',
    category: 'Stylistic Issues',
    description: 'require or disallow spaces inside parentheses',
    eslintUrl: 'http://eslint.org/docs/rules/space-in-parens',
    provider: 'tslint-eslint-rules'
  },
```

The usage section is unnecessary since we have that data in the rule metadata.